src/Compiler.php

```diff
@@ -402,29 +417,15 @@ private function getRootNode(DOMElement $element): \DOMNode
         return $firstTagNode;
     }
 
-    protected function sanitizeCondition(string $condition)
-    {
-        $condition = str_replace('&&', 'and', $condition);
-        $condition = str_replace('||', 'or', $condition);
-        $condition = str_replace('!==', '!=', $condition);
```

Comment:
I learned today that Twig has support for the `A === B` operator, but calls it `A same as(B)`. Maybe we can implement that as well, then we don't miss out on strictness?